index.html

```diff
@@ -0,0 +1,208 @@
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Lato:wght@300;400&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Gothic+A1&family=Inter&family=Lato:wght@300;400&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+rel="stylesheet">
+    
+    
+    <title>Electromagnetics Summit 2023</title>
+</head>
+<body>
+    
+
+    <nav class="desktop navbar navbar-dark bg-dark" >
+        <div class="container text-white">
+            <div class="navbar-brand"></div>
+            <ul class="d-flex " style="margin:0">
+                
+                <li class="nav-item">
+                  <a class="nav-link active text-white" aria-current="page" href="#"><img src="./images/facebook.png"> </a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link text-white" href="#"><img src="./images/twitter.png"></a>
+                </li>
+                <li class="nav-item ">
+                  <a class="nav-link text-white" href="#">English</a>
+                </li>
+                <li class="nav-item ">
+                    <a class="nav-link text-white" href="#">My page</a>
+                </li>
+
+                <li class="nav-item ">
+                    <a class="nav-link text-white" href="#">Log out</a>
+                </li>
+
+            </ul>
+        </div>
+    </nav>
+
+    <nav class="desktop navbar navbar-dark bg-white">
+        <div class="container text-white d-flex justify-content-between">
+            <a href="./index.html" class="navbar-brand"><img src="./images/my_logo.png"></a>
+            <a href="./about.html" class="navbar-brand" id="about" style="color: red" >About</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >Program</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >Join</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >Sponsor</a> 
+            <a href="#" class="navbar-brand" id="about" style="color: black" >News</a>
+            <a href="#" class="navbar-brand" id="about" style="color: red; border: solid 3px red; "  >CC Campaign</a>
+            
+        </div>
+    </nav>
+
+    <nav class="mobile navbar navbar-dark bg-white">
+        <div class="container text-white d-flex justify-content-between">
+            <a href="#" class="navbar-brand" id="hambuger"><img src="./images/mobile_menu.png"></a> 
+            <a href="./index.html" class="navbar-brand"><img src="./images/my_logo.png"></a>
+        </div>
+    </nav>
```

Comment:
- Kindly you are having 3 `nav` which is not recommended to have for semantic reasons please collect them in one `nav`👌👌 and it is better to rap the `nav` with the `header` tag for more readability👌👍